tools/unicode-update.py

```
@@ -0,0 +1,203 @@
+def get_file_hash(file_path: str) -> str | None:
+    """Calculate SHA256 hash of a file."""
+    if not os.path.exists(file_path):
+        return None
+    with open(file_path, 'rb') as f:
+        return hashlib.sha256(f.read()).hexdigest()
+
+
+def download_file(url: str, filename: str) -> str:
+    """Download file and return its hash."""
+    print(f"Downloading {url}...")
+    response = requests.get(url)
```

Ruff (0.14.1)

Probable use of `requests` call without timeout

(S113)